pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -131,6 +131,7 @@ enum ProtocolVersion {
 	v8 = 8;  // Added CommandConsumerStats - Client fetches broker side consumer stats
 	v9 = 9;  // Added end of topic notification
 	v10 = 10;// Added proxy to broker
+	v11 = 11;// Added more security to proxy lookup requests
```

Comment:
Do we need to increase version? If the broker doesn't know about `original_principal` it will just ignore the field. 
The version should only be incremented when we add new Command or features that require checking that the other side supports them.